test/unit/common_tests.py

```diff
@@ -261,6 +262,57 @@ def test_renders_cheetah_missing_result(self, mock_tempfile, mock_run_command, m
             self.assertEquals(call("temp_pickle"), mock_unlink.mock_calls[0])
 
 
+class CargoTransformTest(unittest.TestCase):
+    def setUp(self):
+        self.config_file = NamedTemporaryFile(delete=False).name
+
+    def tearDown(self):
+        os.unlink(self.config_file)
+
+    def test_simple_case(self):
+        cargo_toml = dedent("""
+        [package]
+        name = "hello_world" # the name of the package
+        version = "0.1.0"    # the current version, obeying semver
+        authors = ["you@example.com"]
+        """)
+        with open(self.config_file, 'w') as f:
+            f.write(cargo_toml)
+
+        CargoTagger.tag_new_version("2.2.2-1", self.config_file)
+        output = open(self.config_file, 'r').readlines()
+
+        self.assertEquals(5, len(output))
+        self.assertEquals("[package]\n", output[1])
+        self.assertEquals("name = \"hello_world\" # the name of the package\n", output[2])
+        self.assertEquals("version = \"2.2.2\"    # the current version, obeying semver\n", output[3])
+        self.assertEquals("authors = [\"you@example.com\"]\n", output[4])
+
+    def test_complicated_case(self):
+        cargo_toml = dedent("""
+        [package]
+        name = "hello_world"
+        version = "2.2.2"
+        authors = ["you@example.com"]
+
+        [dependencies]
+        regex = "1.0.0"
+
+        [dependencies.termion]
+        version = "0.1.0"
+        """)
+        with open(self.config_file, 'w') as f:
+            f.write(cargo_toml)
+
+        CargoTagger.tag_new_version("3.3.3-1", self.config_file)
+        output = open(self.config_file, 'r').readlines()
+
+        self.assertEquals(11, len(output))
+        self.assertEquals("version = \"3.3.3\"\n", output[3])
+        self.assertEquals("regex = \"1.0.0\"\n", output[7])
+        self.assertEquals("version = \"0.1.0\"\n", output[10])
+
+
```

Comment:
Could I get one more test going through the entire spec file + version tagger workflow, with a rust Config.toml in the working dir as well? Just want to make absolutely sure we can keep this working for you amidst future changes.